patches/master_patch.patch

```diff
@@ -1571,21 +1607,36 @@ index 08d8639f6fd230eaf5fae1e0a94de504b5c76f2f..c1986c9647249d97f3c5dccf48fe90f6
  @property(readonly) NSTimeInterval timestamp;
  
  @end
+diff --git a/content/browser/renderer_host/plugin_registry_impl.cc b/content/browser/renderer_host/plugin_registry_impl.cc
+index ec071f3b89c16febad73f77a1b17fe864874e371..6c1c79a6d4276d7cf9af7dbc5285dfe1435f313c 100644
+--- a/content/browser/renderer_host/plugin_registry_impl.cc
++++ b/content/browser/renderer_host/plugin_registry_impl.cc
+@@ -55,6 +55,9 @@ void PluginRegistryImpl::GetPluginsComplete(
+   PluginServiceFilter* filter = PluginServiceImpl::GetInstance()->GetFilter();
+   std::vector<blink::mojom::PluginInfoPtr> plugins;
+ 
++
++//  const int child_process_id = render_process_id_;
```

Comment:
why would we comment this out? This is exactly the change we made in the patch above

Reply:
Owp, it was unfinished. It was pending to make available there the `render_process_id_` which wasn't after the merging it with the rebased C69.